docs/tests/visdiff.js

```diff
@@ -0,0 +1,188 @@
+    });
+
+    it("should match List page against golden directory", () => {
+      return takeAndCompareScreenshot(page, "component/list", "wide");
+    });
+
+    it("should match Menu page against golden directory", () => {
+      return takeAndCompareScreenshot(page, "component/menu", "wide");
+    });
+
+    it("should match Radio page against golden directory", () => {
+      return takeAndCompareScreenshot(page, "component/radio", "wide");
+    });
+
+    it("should match Select page against golden directory", () => {
+      return takeAndCompareScreenshot(page, "component/select", "wide");
+    });
+
+    it("should match Slider page against golden directory", () => {
+      return takeAndCompareScreenshot(page, "component/slider", "wide");
+    });
+
+    it("should match Snackbar page against golden directory", () => {
+      return takeAndCompareScreenshot(page, "component/snackbar", "wide");
+    });
+
+    it("should match Switch page against golden directory", () => {
+      return takeAndCompareScreenshot(page, "component/switch", "wide");
+    });
+
+    it("should match Tabs page against golden directory", () => {
+      return takeAndCompareScreenshot(page, "component/tabs", "wide");
+    });
+
+    it("should match Textfield page against golden directory", () => {
+      return takeAndCompareScreenshot(page, "component/textfield", "wide");
+    });
+
+    it("should match Theme page against golden directory", () => {
+      return takeAndCompareScreenshot(page, "component/theme", "wide");
+    });
+
+    it("should match Toolbar page against golden directory", () => {
+      return takeAndCompareScreenshot(page, "component/toolbar", "wide");
+    });
+  });
+});
+
+// - page is a reference to the Puppeteer page.
+// - route is the path you're loading, which I'm using to name the file.
+// - filePrefix is either "wide" or "narrow", since I'm automatically testing both.
+async function takeAndCompareScreenshot(page, route, filePrefix) {
+  // If you didn't specify a file, use the name of the route.
+  let fileName = filePrefix + '/' + (route ? route : 'index');
+
+  // Start the browser, go to that page, and take a screenshot.
+  await page.goto(`http://localhost:8080/${route}`, {
+    waitUntil: route === "component/card" ? "networkidle0" : "load" //card has images to load
+  });
+  await page.screenshot({path: `${testDir}/${fileName}.png`});
+  // Test to see if it's right.
+  return compareScreenshots(fileName);
+}
+
+function compareScreenshots(fileName) {
+  return new Promise((resolve, reject) => {
+    const img1 = fs.createReadStream(`${testDir}/${fileName}.png`).pipe(new PNG()).on('parsed', doneReading);
+    const img2 = fs.createReadStream(`${goldenDir}/${fileName}.png`).pipe(new PNG()).on('parsed', doneReading);
+
+    console.log(base64Img.base64Sync(`${testDir}/${fileName}.png`));
+    console.log("______________________");
+    console.log(base64Img.base64Sync(`${goldenDir}/${fileName}.png`));
```

Comment:
Printing `base64` to console kills Travis and makes it quite unreadable.

Reply:
oh no no this will be reverted, its just a temporary thing